### DISCUSSION TITLE
PHP Storm autocompletion

### DISCUSSION THREAD

**Discussion by Healyhatman** [tighten/ziggy#449] [Q&A]
How do I get PHP Storm to stop telling me that route() doesn't exist in code inspections?

**Comment by anburocky3** [comment]
@Healyhatman  have you found any solution?

**Reply by rossalexander** [reply]
I've been trying to fix this for several hours. Did you happen to figure this out?

**Reply by Healyhatman** [reply]
No I just ignore it

**Reply by anburocky3** [reply]
But in the IDE, it shows error.  How can i avoid that?
